exercises/hello-world/README.md

```diff
@@ -1,6 +1,6 @@
 # Hello World
 
-The classical introductory exercise. Just say "Hello, World!".
+The classic introductory exercise. Just say "Hello, World!".
```

Comment:
Hmm. For classical vs. classic,  I'm leaning toward classical. I must defer to original [description file](https://github.com/exercism/problem-specifications/blob/master/exercises/hello-world/description.md). If a change were made, that repo is where it would be made. The configlet tool generates README.md.

Reply:
fine by me…

I [looked it up](https://jakubmarian.com/classic-vs-classical-in-english/) but even their description was confused!